TODO

```diff
@@ -0,0 +1,8 @@
+
+Todo:
+  ✔ Add a todo
+  ✔ A Basic Rewrite of library in TypeScript
+  ✔ A Basic Rewrite of tests in TypeScript
+  ☐ Add examples for nestjs, expressjs, koa and others
+  ☐ Add Migration guide
+  ☐ Review all Docs
```

Comment:
Would this really be part of the repository, or is it just a list of "personal todos?"